public/app/plugins/panel/table/module.tsx

```diff
@@ -131,6 +131,13 @@ export const plugin = new PanelPlugin<PanelOptions, TableFieldOptions>(TablePane
         defaultValue: [ReducerID.sum],
         showIf: (cfg) => cfg.footer?.show,
       })
+      .addBooleanSwitch({
```

Comment:
Looks good! I'm thinking we should add a description here and maybe change the name of this option as well to make it more clear.

Reply:
And I'd say we might to reverse the language, like "Count Per Field" with a description along the lines of "Count the non-empty results of each field separately"

Reply:
From this I understand we also reverse the functionality. E.g: We show a count for all fields in one column and when the "Count Per Field" switch is active we show a count for each field. This would change all existing table panels for the users. Is this something we'd want?